plugins/modules/helm_repository.py

```diff
@@ -72,6 +72,31 @@
     default: false
     type: bool
     version_added: 2.3.0
+  host:
+    description:
+    - Provide a URL for accessing the API. Can also be specified via C(K8S_AUTH_HOST) environment variable.
+    type: str
+    version_added: "2.2.4"
```

Comment:
These should all be 2.3.0. This is a new feature.

Reply:
I forgot to commit 😉 lol